lib/tasks/scrape_news.rake

```diff
@@ -16,19 +16,53 @@ namespace :scrape_news do
 
     newArray = []
     countArticles = 0
-    counter = 0
+    whichArticle = 0
```

Comment:
In ruby, you should use underscores, instead if camelCase for all variable and function names